addons/web_tour/static/src/js/tip.js

```diff
@@ -193,7 +197,7 @@ var Tip = Widget.extend({
             if (cssPosition === "static" || cssPosition === "relative") {
                 $location.addClass("o_tooltip_parent");
             }
-            this.renderElement();
+            this.$el.detach();
             this.$el.appendTo($location);
```

Comment:
detach then appendTo, so detach is useless...

Reply:
Why there is detach() then appendTo() in bubble_info_mode ? I did it to see if there was a change, none seen, so yeah, I should remove it :)

Reply:
Because there is a line between that detach and that appendTo...